### ISSUE TITLE
"You must choose a company before you can access the "User Login" table." - Since BC19

### ISSUE THREAD

**Issue by CptBlaubart** [MicrosoftDocs/dynamics365smb-devitpro-pb#2248]
I didn't have this issue with my last upgrade to BC18.6 so I assume it's new for BC19. (BC19.2 CH localization to be precise)

When i try to remove the User i had to create for the upgrade (we are upgrading databases that do not belog into our AD), i get rejected by this message:

```
PS C:\Run> Remove-NAVServerUser -ServerInstance BC -UserName UpgUser

MicrosoftDynamicsNavServer$BC
Are you sure you want to remove the user identified by 'UpgUser'?
[Y] Yes  [N] No  [?] Hilfe (Standard ist "Y"): y
You must choose a company before you can access the "User Login" table.
    + CategoryInfo          : NotSpecified: (0:Int32) [Remove-NAVServerUser], FaultException`1
    + FullyQualifiedErrorId : MicrosoftDynamicsNavServer$BC,Microsoft.Dynamics.Nav.Management.Cmdlets.RemoveNavServerU
   ser
```
I don't find any option to provide a company. Nor do i think removing a NavServerUser should be company dependant.

I'm not quite sure if this is an error in the documentation or the feature itself. - However I do need more information on the topic, so... it surely isn't wrong to ask here.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0847ea55-f553-4fff-9ec6-b387365768ed
* Version Independent ID: 519a160a-0595-e40a-0e75-b66b2c983a94
* Content: [Remove-NAVServerUser (Microsoft.Dynamics.Nav.Management) - Dynamics NAV](https://docs.microsoft.com/en-us/powershell/module/microsoft.dynamics.nav.management/remove-navserveruser?view=businesscentral-ps-19)
* Content Source: [businesscentral-ps-19/Microsoft.Dynamics.Nav.Management/Remove-NAVServerUser.md](https://github.com/MicrosoftDocs/nav-docs-powershell/blob/live/businesscentral-ps-19/Microsoft.Dynamics.Nav.Management/Remove-NAVServerUser.md)
* Product: **dynamics365-business-central**
* GitHub Login: @jswymer

**Comment by jswymer** [comment]
Hello,

Thanks for the feedback. It seems that something has changed. We'll look into it and will get back to you.

**Comment by jswymer** [comment]
Hello again,

This turned out to be a bug and has been fixed in update 19.6 and 20.0. Thanks again.
